pkg/source/source.go

```diff
@@ -132,6 +136,10 @@ type Channel struct {
 	destLock sync.Mutex
 }
 
+func (c *Channel) String() string {
+	return fmt.Sprintf("channel source: %p", c)
```

Comment:
@DirectXMan12 @droot I chose to print the pointer because I didn't know of a better field to make it identifiable and thoughts?

Reply:
hmm..can't think of better alternative in this case.